cocinaborcelle/src/components/CardCompras.tsx

```diff
@@ -0,0 +1,49 @@
+
+import { useState } from 'react';
+import styles from "@/styles/CardCompras.module.css";
+
+type CardComprasProps = {
+    title?: string;
+    number: number;
+}
```

Comment:
```suggestion
import { Comida, Compra } from '@prisma/client';

type CardCompraProps = {
    compra: Compra & {comida: Comida}
}
```

seria bom vc passar algo que é possível retornar da api para este card, como uma compra e a comida atrelada a esta compra, que aí o componente vai receber nessa prop compra todas as informações que precisa para formar o card